app/components/OrderItem/index.js

```diff
@@ -80,56 +64,52 @@ export default class OrderItem extends Component {
         onPress={this.props.onPress}
       >
         <View style={{ flex: 1, flexDirection: 'row' }}>
-          <Grid style={{ flex: 9 }}>
+          <Grid style={{ flex: 1 }}>
             <Col style={styles.left}>
               <Text style={styles.orderId}>Order-{this.props.order.id}</Text>
               <Text
                 note
                 style={styles.orderId}
               >
-                {this.formatDate(this.props.order.created_at)}
+                {formatDate(this.props.order.created_at)}
               </Text>
             </Col>
-            <Col style={styles.center}>
-              <Text>{Intl.NumberFormat('id').format(this.props.order.amount)}</Text>
+            <Col style={styles.right}>
+              <Text style={styles.text}>{Intl.NumberFormat('id').format(this.props.order.amount)}</Text>
```

Comment:
Can I know what does this `Intl` do? it seems to be the source of crashes on my device.

Reply:
Intl = International formating, seems like it used for automatic text formating. It already solved now, i include the reason below